tests/virt_autotest/login_console.pm

```diff
@@ -128,7 +128,15 @@ sub login_to_console {
             send_key 'down';
             send_key 'ret';
             #wait sshd up
-            assert_screen('sshd-server-started', 180);
+            my $remote_machine = get_required_var('SUT_IP');
+            record_info("Will detect whether machine $remote_machine boots up and has ssh port open", "Calling virt_autotest::utils::check_port_state");
```

Comment:
Please see my comment above regarding the length of the string in `record_info`

Reply:
Done